src/VisualStudio/Core/Def/ProjectSystem/MetadataReferences/VisualStudioPortableExecutableReference.cs

```diff
@@ -28,32 +29,32 @@ internal partial class VisualStudioMetadataReferenceManager
     /// instance of <see cref="VisualStudioPortableExecutableReference"/> is created for the corresponding reference.
     /// </remarks>
     [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
-    private sealed class VisualStudioPortableExecutableReference : PortableExecutableReference, ISupportTemporaryStorage
+    private sealed partial class VisualStudioPortableExecutableReference : PortableExecutableReference, ISupportTemporaryStorage
     {
         private readonly VisualStudioMetadataReferenceManager _provider;
         private readonly Lazy<DateTime> _timestamp;
-        private readonly FileChangeTracker? _fileChangeTracker;
+        private readonly IFileChangeWatcher _fileChangeWatcher;
 
         private Exception? _error;
 
+        private static readonly FileTimeStampProvider s_timeStampProvider = new();
+
         internal VisualStudioPortableExecutableReference(
             VisualStudioMetadataReferenceManager provider,
             MetadataReferenceProperties properties,
             string fullPath,
-            FileChangeTracker? fileChangeTracker)
```

Comment:
>fileChangeTracker [](http://example.com/codeflow?start=31&length=17)

This was always null, so the FileChangeTracker class ended up not being used anywhere.

Reply:
I had long been wanting to delete that...didn't realize it was that close to being dead.

Reply:
Should we just be deleting the field then at that point? Or do we still need to eventually figure out the subscription guarantee?

Reply:
The field was needed due to it's use in WithPropertiesImpl.

Reply:
Oh I see: the old thing wasn't using it, but now the replacement is being used.